internal/characters/eula/skill.go

```diff
@@ -179,14 +179,14 @@ func (c *char) holdSkill(p map[string]int) action.ActionInfo {
 	if v > 0 {
 		done := false
 		shredCB = func(a combat.AttackCB) {
-			if done {
-				return
-			}
+			
 			e, ok := a.Target.(*enemy.Enemy)
 			if !ok {
 				return
 			}
-			done = true
+			if e.Type() != targets.TargettableEnemy {
+				return
+			}
```

Comment:
i don't think this is necessary, since there's a casting to `*enemy.Enemy` above.